xcodebuild/build.go

```diff
@@ -42,78 +51,96 @@ type ArchiveCommandModel struct {
 	forceProvisioningProfileSpecifier string
 	forceProvisioningProfile          string
 	forceCodeSignIdentity             string
+	disableCodesign                   bool
 
 	// buildaction
 	customBuildActions []string
 
 	// Options
 	archivePath   string
 	customOptions []string
+	sdk           string
+
+	// Archive
+	buildCommand BuildCommand
 }
 
-// NewArchiveCommand ...
-func NewArchiveCommand(projectPath string, isWorkspace bool) *ArchiveCommandModel {
-	return &ArchiveCommandModel{
-		projectPath: projectPath,
-		isWorkspace: isWorkspace,
+// NewBuildCommand ...
+func NewBuildCommand(projectPath string, buildCommand BuildCommand, isWorkspace bool) *BuildCommandModel {
```

Comment:
- NewCommand ? 
- keep the `isWorkspace` close to the `projectPath ` since they are related